clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5532,6 +5532,23 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
 
     if (Arg *A = Args.getLastArg(options::OPT_fsycl_exp_range_rounding))
       A->render(Args, CmdArgs);
+    if (Arg *A = Args.getLastArg(options::OPT_fsycl_fp64_conv_emu)) {
```

Comment:
We should emit the warning when we gather all of the Triples.  See similar usage of emitting a warning for `-fno-sycl-libspirv` in `clang/lib/Driver.cpp:1268`.  Also, rendering the option should be based on `IsSPIRAOT() && SPIRSubArch_gen`